tests/TestCase/ToolbarServiceTest.php

```diff
@@ -73,6 +73,25 @@ public function testLoadPanels()
         $this->assertGreaterThan(1, $this->events->listeners('Controller.shutdown'));
         $this->assertInstanceOf('DebugKit\Panel\SqlLogPanel', $bar->panel('SqlLog'));
     }
+    
```

Comment:
Can you fix this please? Make sure your IDE has trim right enabled to not pollute the files with whitespace noise.